packages/react-checkbox/src/components/Checkbox/Checkbox.types.ts

```diff
@@ -9,7 +9,10 @@ import { LabelProps } from '@fluentui/react-label';
 
 export interface CheckboxCommons {
   /**
-   * A checkbox can be rendered with a circular shape.
+   * Whether to render the checkbox in a circular shape instead of square.
+   * This variant is only recommended to be used in a tasks-style UI (checklist),
+   * since it otherwise could be confused for a `RadioItem`.
+   * @defaultvalue false
    */
   circular: boolean | undefined;
```

Comment:
Nit: circular?: boolean 

(see my other comment about if we should have shape?: 'square' | 'circular'

Reply:
Changed to `circular: boolean`. It's not optional on the state, so I didn't add `?`, and instead gave it a default value of `false` in `useCheckbox`.